eloquentjs/11_1_quiet_times/quiet_times.js

```diff
@@ -0,0 +1,47 @@
+const { generateLogs } = require('./src/generateLogs');
+const { textFile } = require('./src/textFile');
+
+/**
+ * Returns promise of hourly security camera activity recorded for given day of week (0-6)
+ * @param {*} day 
+ * @returns 
+ */
+async function activityTable(day) {
+    const camera_logs = await textFile('camera_logs.txt');
+    const filePaths = camera_logs.split('\n').filter(j => j);
+
+    const timeStamps = [];
+    for (let i = 0; i < filePaths.length; i++) {
+        const logContents = await textFile(`weekly_logs/${filePaths[i]}`);
+
+        logContents.split('\n').filter(j => j).forEach(element => {
+            timeStamps.push(element);
+        });
+    }
+
+    const activity = Array(24).fill(0);
+
+    timeStamps.forEach((time) => {
+        const date = new Date();
```

Comment:
I'm not 100% sure on this, but I think you can instantiate the Date object correctly with `new Date(time)`

Reply:
I had tried that and gotten a bug but it turns out I had been trying to feed it mS as a string. Whoops. It does seem inconsistent though that Dates setTime method does accept mS values as strings. 🤷‍♂️ Patched for next push.

Reply:
It's very hard for software that makes design mistakes to go back and fix it later because doing so breaks existing programs. This reads to me like a historical accident